src/gpodder/gtkui/main.py

```diff
@@ -3740,12 +3735,24 @@ def check_for_updates(self, silent):
                 _('Installed version: %s') % gpodder.__version__,
                 _('Newest version: %s') % version,
                 _('Release date: %s') % released,
-                '',
-                _('Download the latest version from gpodder.org?'),
             ])
+            if os.path.exists(gpodder.no_update_check_file):
+                message = '\n'.join([
+                    message,
+                    '',
+                    _('Please check your distribution for gPodder updates.'),
```

Comment:
Is "your distribution" clear to all users? Or should it be something like "your system's package manager"?